tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/charts/Course_Info_e0098c.yaml

```diff
@@ -0,0 +1,224 @@
+_file_name: Course_Info_e0098c.yaml
+cache_timeout: null
+certification_details: null
+certified_by: null
+dataset_uuid: 27a4476b-5a9a-4fc4-ac75-da520b522341
+description: null
+params:
+  adhoc_filters:
+  - clause: WHERE
+    comparator: No filter
+    expressionType: SIMPLE
+    operator: TEMPORAL_RANGE
+    subject: emission_time
+  all_columns: []
+  color_pn: false
+  column_config:
+    Current Enrollees:
+      columnWidth: 150
+    active_count:
+      columnWidth: 200
+    course_name:
+      columnWidth: 200
+    course_run:
+      columnWidth: 300
+    tag_list:
+      truncateLongCells: true
+  conditional_formatting: []
+  extra_form_data: {}
+  groupby:
+  - course_run
+  - course_name
+  metrics:
+  - aggregate: COUNT_DISTINCT
+    column:
+      advanced_data_type: null
+      certification_details: null
+      certified_by: null
+      column_name: actor_id
+      description: null
+      expression: null
+      filterable: true
+      groupby: true
+      id: 7901
+      is_certified: false
+      is_dttm: false
+      python_date_format: null
+      type: String
+      type_generic: 1
+      verbose_name: null
+      warning_markdown: null
+    datasourceWarning: false
+    expressionType: SIMPLE
+    hasCustomLabel: true
+    label: Current Enrollees
+    optionName: metric_j4kvubgy7m_afs23ho378e
+    sqlExpression: null
+  - active_count
+  - tag_list
```

Comment:
I'm not 100% sure but I think metrics should be added to the datasets to allow them to be translated

Reply:
ah i updated 1 of the charts but not the other. thanks!